lib/utils/report.mjs

```diff
@@ -43,7 +43,7 @@ export default function report(problem) {
 	const startLine = line || (start && start.line);
 
 	if (!startLine) {
-		throw new Error('You must pass either a node or a line number');
+		throw new Error(`You must pass either a node or a line number (rule: ${ruleName})`);
```

Comment:
Am I right in thinking that only the (custom) rule author can resolve this issue and not the user? If so, let's rewrite the error to address the original problem: lack of clarity around the message.

```suggestion
		throw new Error(`The "${ruleName}" (custom) rule failed to pass either a node or a line number to the `report()` function.`);
```

Reply:
Nice catch! Your suggestion is much better. 👍🏼